bindings/go/oci/repository.go

```diff
@@ -134,7 +137,16 @@ func (repo *Repository) ListComponentVersions(ctx context.Context, component str
 			Subject:         indexv1.Descriptor,
 			VersionResolver: complister.ReferrerAnnotationVersionResolver(component),
 		},
-	})
+	}
+
+	switch repo.referrerTrackingPolicy {
+	case ReferrerTrackingPolicyByIndexAndSubject:
+		opts.LookupPolicy = lister.LookupPolicyReferrerWithTagFallback
+	case ReferrerTrackingPolicyNone:
+		opts.LookupPolicy = lister.LookupPolicyTagOnly
+	}
```

Comment:
Default case in which the setting is uninitialized?

Reply:
default case would be zero-value which is ReferrerTrackingPolicyNone